CONTRIBUTING.md

```diff
@@ -193,30 +276,56 @@ controlPlane:
       tag: 0.0.1
 ```
 
-Then deploy the vCluster with the provided `vcluster.yaml`:
+#### Deploy vCluster 
+
+Launch your vCluster using your `vcluster.yaml`
+
+```
+$ vcluster create my-vcluster -n my-vcluster -f ./vcluster.yaml --local-chart-dir chart
+```
```

Comment:
if you build the cli with `just build-snapshot` the charts are embedded in it already, so you don't even need to use `--use-local-chart-dir`